ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1065,7 +1065,12 @@ IopEnumerateDetectedDevices(
    UNICODE_STRING HardwareIdKeyboard = RTL_CONSTANT_STRING(L"*PNP0303\0");
    static ULONG DeviceIndexKeyboard = 0;
    const UNICODE_STRING IdentifierMouse = RTL_CONSTANT_STRING(L"PointerController");
+   // FIXME: IopEnumerateDetectedDevices() should be rewritten.
```

Comment:
Suggestion to make this comment more detailed, based on [this message](https://chat.reactos.org/reactos/pl/5e1xra5hqpgx88a11a6phgbmsh).
```suggestion
   /* FIXME: IopEnumerateDetectedDevices() enumerates legacy devices using hardcoded PnP identifiers.
    * It should be rewritten by implementing an actual parser of LegacyXlate sections of driver INF files. */
```